op-conductor/conductor/service.go

```diff
@@ -491,6 +521,23 @@ func (oc *OpConductor) HTTPEndpoint() string {
 
 func (oc *OpConductor) OverrideLeader(override bool) {
 	oc.leaderOverride.Store(override)
+
+	if oc.rollupBoost != nil {
+		mode := false
+		if override {
+			mode = true
+		}
+
+		// Use a background context since this is an async operation
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		if err := oc.rollupBoost.SetExecutionMode(ctx, mode); err != nil {
+			oc.log.Error("Failed to update rollup boost execution mode after override", "err", err, "mode", mode)
```

Comment:
should we set leaderOverride back to false if rollupboost execution_mode is not able to be enabled?